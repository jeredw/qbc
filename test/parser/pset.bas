pset step(0,0)
pset step(0,0), 15
pset (15, 10)
pset (15, 10), 15
