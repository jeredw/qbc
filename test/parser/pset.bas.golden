(program 
  (statement 
    (pset_statement pset step 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ))) \n 
  (statement 
    (pset_statement pset step 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) , 
      (expr 
        (literal 15)))) \n 
  (statement 
    (pset_statement pset 
      ( 
        (expr 
          (literal 15)) , 
        (expr 
          (literal 10)) ))) \n 
  (statement 
    (pset_statement pset 
      ( 
        (expr 
          (literal 15)) , 
        (expr 
          (literal 10)) ) , 
      (expr 
        (literal 15)))) \n <EOF>)
